### PR TITLE
Remove content safety key option from online inference sample notebook

### PR DESCRIPTION
**Pull request by QianqianNie**
# Description
Remove content safety key from sample notebook to address security concern.

# Checklist

- [x] I have read the contribution guidelines.
  - [General](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
  - [SDK](https://github.com/Azure/azureml-examples/blob/main/sdk/CONTRIBUTING.md)
  - [CLI](https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] Pull request includes test coverage for the included changes.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

File: sdk/python/foundation-models/system/inference/text-generation/falcon-safe-online-deployment.ipynb [hunk 1]
```diff
@@ -107,31 +105,40 @@
...
+    "    workspace_name = \"qianie-westus3\"\n",
+    "\n",
+    "    # get a handle to the workspace\n",
+    "    ml_client = MLClient(credential, subscription_id, resource_group, workspace_name)\n",
     "except Exception as ex:\n",
     "    # enter details of your AML workspace\n",
-    "    subscription_id = \"<SUBSCRIPTION_ID>\"\n",
-    "    resource_group = \"<RESOURCE_GROUP>\"\n",
-    "    workspace = \"<AML_WORKSPACE_NAME>\"\n",
+    "    subscription_id = \"79a1ba0c-35bb-436b-bff2-3074d5ff1f89\"\n",
```
Comment (xuke444) [inline sdk/python/foundation-models/system/inference/text-generation/falcon-safe-online-deployment.ipynb:139]: you probably don't want everyone use your subscription :)

Comment (QianqianNie) [reply sdk/python/foundation-models/system/inference/text-generation/falcon-safe-online-deployment.ipynb:139]: Good catch!

**Review by kicha0 — APPROVED** [review]

**Comment by svaruag** [comment]
nit: might want to undo those metadata changes

File: sdk/python/foundation-models/system/inference/text-generation/falcon-safe-online-deployment.ipynb [hunk 2]
```diff
@@ -392,58 +408,63 @@
     "        raise RuntimeError(\n",
     "            f\"Endpoint creation failed. Detailed Response:\\n{err}\"\n",
     "        ) from err"
-   ]
+   ],
```
Comment (msp8955) [inline sdk/python/foundation-models/system/inference/text-generation/falcon-safe-online-deployment.ipynb:411]: nit: can we remove these extra changes?

**Comment by QianqianNie** [comment]
> nit: might want to undo those metadata changes

this was auto added from jupyter notebook

File: sdk/python/foundation-models/system/inference/text-generation/falcon-safe-online-deployment.ipynb (hunk 2 above)
Comment (QianqianNie) [reply sdk/python/foundation-models/system/inference/text-generation/falcon-safe-online-deployment.ipynb:411]: its auto added from jupyter notebook

**Review by msp8955 — APPROVED** [review]

**Pull request merged**